ansible/roles/common/tasks/main.yml

```diff
@@ -39,6 +40,17 @@
     - gcc
     - python-pip
     - open-iscsi
+  when: ansible_distribution == "Ubuntu" and ansible_lsb.major_release|int >=16
+
+- name: Install system packages on CentoS or RHEL
+  when: (ansible_distribution == "CentOS" or ansible_distribution == "RedHat")
```

Comment:
Can we add support version check (ansible_distribution_major_version?) or document it some where?

Reply:
Done

Reply:
Done